views/footer.pug

```diff
@@ -0,0 +1,7 @@
+
+div.footer-nav
+  a(href= '/about') About
+  a(href= '/team') Team
+  a(href= '/contact') Contact
+  a(href= '/users/login') Login
+  p Coypright (c) Dragonboard
```

Comment:
Can we remove this? We're copying something else, don't want to copyright this thing...